index.html

```diff
@@ -65,6 +68,266 @@ <h2 class="my-position">I’am a Software Developer</h2>
+                    <li class="language-list-content">
+                        Codepen
+                    </li>
+                    <li class="language-list-content">
+                        Gitlab
+                    </li>
+                    <li class="language-list-content">
+                        Terminal
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="lastpage">
+        <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
+            please get in touch.</h2>
+        <div class=form>
+            <input class="fullname" type="text" name="fullname" placeholder="Full name">
+            <input class="fullname" type="text" name="fullname" placeholder="Email address">
+            <div class="text-area">
+                <textarea class="textarea-1" placeholder="Enter text here"></textarea>
+            </div>
+            <button type="submit" class="button1 button1-text button-end">Get in toucht</button>
+        </div>
+        <div class="project-footer page-footer">
+            <img alt="Line image" class="footer-line" src="css/image/projects_imges/line3.png">
+            <div class="page-footer">
+                <div class="social-media">
+                    <ul class="icon-list">
+                        <li class="img-list">
+                            <a href="https://github.com/zakiyousufi" target="_blank" rel="noopener">
+                                <img class="github icons" alt="Github Icon" src="css/Icons/social_media/github.png">
+                            </a>
+                        </li>
+                        <li class="img-list">
+                            <a href="#" target="_blank" rel="noopener">
+                                <img class="linkedin icons" alt="LinkedIn Icon"
+                                    src="css/Icons/social_media/linkedin.png">
+                            </a>
+                        </li>
+                        <li class="img-list">
+                            <a href="#" target="_blank" rel="noopener">
+                                <img class="due icons" alt="Social Media Icon" src="css/Icons/social_media/due.png">
+                            </a>
+                        </li>
+                        <li class="img-list">
+                            <a href="#" target="_blank" rel="noopener">
+                                <img class="tweeter icons" alt="Tweeter Icon" src="css/Icons/social_media/tweeter.png">
+                            </a>
+                        </li>
+                        <li class="img-list">
+                            <a href="#" target="_blank" rel="noopener">
+                                <img class="m-icon icons" alt="Social Media Icon" src="css/Icons/social_media/M.png">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Kindly align center the icons. You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Design
![Screenshot from 2022-05-28 20-06-37](https://user-images.githubusercontent.com/77842237/170835522-79c42bf0-1353-41dd-935c-db65f66bcc85.png)

Figma Design

![Screenshot from 2022-05-28 20-07-07](https://user-images.githubusercontent.com/77842237/170835534-bef37912-3ad2-4f7d-8902-d651fcced35c.png)